lib/content-services/src/lib/search/components/search-filter-chips/search-facet-chip-tabbed/search-facet-chip-tabbed.component.spec.ts

```diff
@@ -0,0 +1,232 @@
+    fixture.detectChanges();
+  }
+
+  function triggerComponentChanges() {
+    component.ngOnChanges({
+        tabbedFacet: new SimpleChange(null, component.tabbedFacet, false)
+      });
+    fixture.detectChanges();
+  }
+
+  function addBucketItem(field: string, displayValue: string) {
+    component.tabbedFacet.facets[field].buckets.items.push({ count: 1, label: displayValue, display: displayValue, filterQuery: '' });
+    triggerComponentChanges();
+  }
+
+    it('should display correct label for tabbed facet', () => {
+        const label = fixture.debugElement.query(By.css('.adf-search-filter-ellipsis')).nativeElement.innerText;
+        expect(label).toBe(component.tabbedFacet.label + ':');
+    });
+
+    it('should display any as display value when nothing is selected', () => {
+        const displayValue = getDisplayValue();
+        expect(displayValue).toBe('SEARCH.FILTER.ANY');
+    });
+
+    it('should display remove icon and disable facet when no items are loaded',  () => {
+        const chip = fixture.debugElement.query(By.css('mat-chip'));
+        const icon = fixture.debugElement.query(By.css('mat-chip mat-icon')).nativeElement.innerText;
+        expect(chip.classes['mat-chip-disabled']).toBeTrue();
+        expect(icon).toEqual('remove');
+    });
+
+    it('should not open context menu when no items are loaded',  () => {
+        spyOn(component.menuTrigger, 'openMenu');
+        const chip = fixture.debugElement.query(By.css('mat-chip')).nativeElement;
+        chip.dispatchEvent(new KeyboardEvent('keyup', { key: 'Enter' }));
+        expect(component.menuTrigger.openMenu).not.toHaveBeenCalled();
+    });
+
+    it('should display correct title when facet is opened', () => {
+        openFacet();
+        const title = fixture.debugElement.query(By.css('.adf-search-filter-title')).nativeElement.innerText.split('\n')[0];
+        expect(title).toBe(component.tabbedFacet.label);
+    });
+
+    it('should display 2 tabs with specific labels', () => {
+        openFacet();
+        const tabLabels = getTabs();
+        expect(tabLabels.length).toBe(2);
+        expect(tabLabels[0].innerText).toBe(component.tabbedFacet.facets['field'].label);
+        expect(tabLabels[1].innerText).toBe(component.tabbedFacet.facets['field2'].label);
+    });
+
+    it('should display creator tab as active initially and allow navigation', () => {
+        openFacet();
+        let activeTabLabel = fixture.debugElement.query(By.css('.mat-tab-label-active .mat-tab-label-content')).nativeElement.innerText;
+        expect(activeTabLabel).toBe(component.tabbedFacet.facets['field'].label);
+
+        changeTab(1);
+        activeTabLabel = fixture.debugElement.query(By.css('.mat-tab-label-active .mat-tab-label-content')).nativeElement.innerText;
+        expect(activeTabLabel).toBe(component.tabbedFacet.facets['field2'].label);
+    });
+
+    it('should display arrow down icon and not disable the chip when items are loaded',  () => {
```

Comment:
here also double space and below as well check please

Reply:
Removed